app/views/lookbook/components/_embed.html.erb

```diff
@@ -1,4 +1,4 @@
-<div id="<%= id %>" x-data="embed" class="not-prose embed border border-gray-300 rounded-md overflow-hidden" @page:morphed.window="recaclulateIframeHeight">
+<div id="<%= id %>" x-data="embed" class="not-prose embed border border-gray-300 rounded-md overflow-hidden" @page:morphed.window="recaclulateIframeHeight" @tab-switch.window="recaclulateIframeHeight">
```

Comment:
Embedded previews on hidden tabs need their height recalculated when visible for the first time (otherwise they hide most of the content)... this event is emitted when tabs are changed.